Create pmdata/news tab

SSZ4 opened the pull request:
미세먼지와 뉴스를 선택해서 볼 수 있는 탭을 만들었습니다.

SSZ4 committed:
Create pmdata/news tab

미세먼지와 뉴스를 선택해서 볼 수 있는 탭을 만들었습니다.

seungyeon119 reviewed (approved):
탭 구분으로 더 깔끔하게 보기 좋은 것 같습니다!

luneclaire commented on src/App.js:
```diff
@@ -18,6 +19,7 @@ function App() {
 
   const [pmSwitch, setPmSwitch] = useState(true)
   const [daySwitch, setDaySwitch] = useState(true)
+  const [tabSwitch, setTabSwitch] = useState(true)
```
tabSwitch라는 변수명은 true일때 어떤 탭이 활성화 되어있는지를 알 수 없어서 불명확한 것 같습니다. true가 news tab이라면 newsTabOn, 혹은 미세먼지 탭이라면 pmTabOn등의 변수명을 사용하는 것이 좋을 것 같습니다.

pmSwitch, daySwitch도 비슷하게 변수명 수정해주면 좋을 것 같습니다!
pmSwitch의 경우, true면 미세먼지, false면 초미세먼지니 pmSwitch -> isPm
daySwitch의 경우 true면 오늘, false면 내일이니 daySwitch -> isToday
등으로 변수명을 확실하게 변경해주면 좋을 것 같습니다.

SSZ4 replied:
네, 좋은 지적 감사합니다. 반영해서 변수명 처리 했습니다!

luneclaire commented on src/SelectTab.js:
```diff
@@ -0,0 +1,16 @@
+		<Radio.Group onChange={onChangeTab} defaultValue="pm" size="large">
+      		<Radio.Button value="pm">미세먼지</Radio.Button>
+      		<Radio.Button value="news">뉴스</Radio.Button>
+    	</Radio.Group>
```
들여쓰기 맞춰주시면 좋을 것 같아요!

SSZ4 replied:
넵 요청사항 모두 수정했습니다.

SSZ4 committed:
Change variable name

직관성을 위해서 요청받은대로
tabSwitch라는 변수명은 true일때 어떤 탭이 활성화 되어있는지를 알 수 없어서 불명확한 것 같습니다. true가 news tab이라면 newsTabOn, 혹은 미세먼지 탭이라면 pmTabOn등의 변수명을 사용하는 것이 좋을 것 같습니다.

pmSwitch, daySwitch도 비슷하게 변수명 수정해주면 좋을 것 같습니다!
pmSwitch의 경우, true면 미세먼지, false면 초미세먼지니 pmSwitch -> isPm
daySwitch의 경우 true면 오늘, false면 내일이니 daySwitch -> isToday
등으로 변수명을 확실하게 변경해주면 좋을 것 같습니다.
를 적용했습니다.

SSZ4 committed:
Change variable name

마찬가지로 직관성을 위해서 변수 이름을 변경했습니다.

SSZ4 committed:
Merge branch 'master' into feature/seungjin/UI

luneclaire commented on src/App.js:
```diff
@@ -100,16 +100,16 @@ function App() {
-          <SelectDay swapPm = {swapPm} swapDay = {swapDay}/>
-          <Map pmSwitch = {pmSwitch} daySwitch = {daySwitch} addr = {addr} changeAddr = {changeAddr} SidoDB = {SidoDB} SigunguDB = {SigunguDB} forecastDB = {forecastDB}/>
+          <SelectDayPm swapIsPm = {swapIsPm} swapIsToDay = {swapIsToDay}/>
+          <Map isPm = {isPm} isToday = {isToday} changeAddr = {changeAddr} SidoDB = {SidoDB} SigunguDB = {SigunguDB} forecastDB = {forecastDB}/>
```
여기 addr = {addr} 없어서 map.js에서 에러 날 것 같아요ㅠㅠ

luneclaire commented on src/Map.js:
```diff
@@ -13,14 +13,14 @@ import { ReactComponent as pin} from './icon/pin.svg'
 import { ColorLegend } from './ColorLegend';
 
-export function Map( {pmSwitch, daySwitch, addr, changeAddr, SidoDB, SigunguDB, forecastDB} ) {
+export function Map( {isPm, isToday, changeAddr, SidoDB, SigunguDB, forecastDB} ) {
```
넘겨받는 props중에 addr이 빠져있고, Map.js내부에 pmSwitch, daySwitch를 사용한 모든 부분에서 새로운 변수명으로 변경해줘야 할 것 같습니다!

SSZ4 committed:
Change value name

SSZ4 committed:
Fix Sido search Error
